### DISCUSSION TITLE
Arlo streaming proxy

### DISCUSSION THREAD

**Discussion by kaffetorsk** [blakeblackshear/frigate#5390] [Show and tell]
I've put together a service that makes it possible to use battery powered Arlo cameras with frigate.

https://github.com/kaffetorsk/arlo-streamer

From description:
Python script that turns arlo cameras into continuous streams through ffmpeg This allow arlo cameras to be used in the NVR of your choosing. (e.g. [Frigate](https://frigate.video/))

The streams will provide an "idle" picture when the camera is not actively streaming. Motion will trigger an active stream, replacing the "idle" picture with the actual camera stream.

Enjoy

**Comment by kendiener** [comment]
Does this work with Arlo 4?
